.github/workflows/pr-checks.yml

```
@@ -8,6 +8,10 @@ on:
   push:
     branches: [ main, develop ]
 
+  permissions:
```

actionlint (1.7.7)

unknown Webhook event "permissions". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)